app/api/v1/endpoints/prediction.py

```
@@ -0,0 +1,48 @@
+    endpoint=settings.PREDICTION_ENDPOINT,
+    credentials=credentials
+)
+
+@router.post("/predict", response_model=List[PredictionResult])
+async def predict_image(request: ImagePredictionRequest):
+    # URL에서 이미지를 다운로드
+    try:
+        response = requests.get(request.image_url)
+        response.raise_for_status()
+    except requests.exceptions.HTTPError as err:
+        raise HTTPException(status_code=400, detail="Error fetching image: " + str(err))
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)